.github/workflows/publish.yml

```diff
@@ -6,12 +6,44 @@ on:
 
 jobs:
   publish:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: [linux/amd64, linux/arm64, android/arm64]
+        runner: [ubuntu-latest, buildjet-8vcpu-ubuntu-2204-arm]
```

Comment:
Can technically remove these lists, but I think it's considered idiomatic to list out all the options in a list like this